search/tests/tests.py

```diff
@@ -625,6 +684,46 @@ def test_datetime_conversion(self):
         json_datetime = "2015-01-31T07:30:28.65785Z"
         self.assertTrue(json_date_to_datetime(json_datetime), datetime(2015, 1, 31, 7, 30, 28, 65785))
 
+    def test_timezone_conversion(self):
```

Comment:
They are not really important (at least not art the moment). In practice all the times are in UTC; the reason why it was needed is that the json serialisation for file-based operation tips the tzinfo - so sometimes we get tests that include it and sometimes not.

This is only the mock that supports this, so I'd rather not add functionality to the mock where it is not necessary.